app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/address/PropertyDetailsController.scala

```diff
@@ -873,31 +873,43 @@ class PropertyDetailsController @Inject() (
                       errorHandler.errorResult()
                     },
                     _ =>
+                      m.triageAnswers.fold(_.numberOfProperties, x => Some(x.numberOfProperties)) match {
+                        case None =>
+                          Redirect(returnsRoutes.TaskListController.taskList())
+                        case Some(propertyCount) =>
+                          Ok(
+                            multipleDisposalsCheckYourAnswersPage(
+                              completeAnswers,
+                              shouldAskIfPostcodeExists(assetTypes),
+                              r.journey.subscribedDetails.isATrust,
+                              extractIndividualUserType(r),
+                              extractDateOfDeath(r),
+                              firstTimeVisiting = true,
+                              addressLine1,
+                              propertyCount,
+                            )
+                          )
+                      }
+                  )
+
+                case c: CompleteExamplePropertyDetailsAnswers =>
+                  m.triageAnswers.fold(_.numberOfProperties, x => Some(x.numberOfProperties)) match {
+                    case None =>
+                      Redirect(returnsRoutes.TaskListController.taskList())
+                    case Some(propertyCount) =>
                       Ok(
                         multipleDisposalsCheckYourAnswersPage(
-                          completeAnswers,
+                          c,
```

Comment:
does this have a proper name?

Reply:
Yes, but this is just an indentation change.